File: FirstBit Workspace/C Programming/Assignments/Assignment 8/Q9.c
//reverse the given array
#include<stdio.h>
void main(){
	int arr[5];
	
	printf("Enter the elements in the array: ");
	for(int i=0;i<5;i++){
		scanf("%d", &arr[i]);
	}
	
	printf("\n\nArray: ");
	for(int i=0;i<5;i++){
		printf("%d ", arr[i]);
	}
	
	printf("\n\nReversed Array: ");
	int end=4;
	for(int i=0;i<end;i++){
		int temp = arr[i];
		arr[i]=arr[end];
		arr[end]=temp;
		end--;
	}
	
	for(int i=0;i<5;i++){
		printf("%d ", arr[i]);
	}
	
}

File: FirstBit Workspace/Core Java/In Class Work/ArrayCProgramsInJava/src/Q9.java
import java.util.Scanner;

public class Q9 {
	
	public static void main(String[] args) {
		
		//reverse the given array
		
		//array reference 
		int[] array;
		
		//initialization
		array = new int[5];
		
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Enter elements to store in array: ");
		
		//store elements
		for(int i=0; i<array.length; i++) {
			array[i] = sc.nextInt(); 
		}
		
		System.out.println("\nElements stored in array: ");
		//display array
		for(int ele:array) {
			System.out.println(ele);
		}
		
		int end = array.length-1;
		
		for(int i=0; i < end; i++) {
			int temp = array[i];
			array[i] = array[end];
			array[end] = temp;
			end--;
		}
		
		System.out.println("\nReversed array: ");
		for(int ele:array) {
			System.out.println(ele);
		}
		
		sc.close();
		
	}

}
